renew

smily342 opened the pull request:
## Netlify link
[Add your Netlify link here.](https://mysmallportfolio243.netlify.app/)
PS. Don't forget to add it in your readme as well.

smily342 committed:
renew

JennieDalgren reviewed (dismissed):
Take a look at the design files closely and make sure your coded version matches the figma version.

smily342 committed:
fixing

smily342 commented:
waiting for viewing

HIPPIEKICK reviewed (changes requested):
Sorry for the late feedback Brian, but amazing progress here ⭐ Just a couple of things left:
- Make sure it's responsive so that there's no side scroll in mobile for example
- Make sure all contrasts are OK. You can use Lighthouse to test this. Tip: test in both dark mode and light mode or at least make sure your portfolio looks the same in dark mode as in light mode.

Almost there 💪 Request a re-review here on GitHub when you're ready.

smily342 committed:
Fix contrast, icon accessibility, prop-types validation

smily342 commented:
Contrast ratio issues in dark mode
Accessibility labels for icon links
went from 84% to 95% accessibiliy on lighthouse @JennieDalgren  @HIPPIEKICK

smily342 commented:
Hey there Mathilda thank you for coming back to me 

We the last change I did this weekend.

Im ready for a new a review

Take care 

Le 2025-05-13T13:23:13.000+02:00, Matilda Brunemalm
***@***.***> a écrit :

>***@***.*** requested changes on this pull request.
> 
> Sorry for the late feedback Brian, but amazing progress here ⭐
> Just a couple of things left:
> 
>      * Make sure it's responsive so that there's no side scroll in
> mobile for example
>      * Make sure all contrasts are OK. You can use Lighthouse to test
> this. Tip: test in both dark mode and light mode or at least make
> sure your portfolio looks the same in dark mode as in light mode.
> 
> Almost there 💪 Request a re-review here on GitHub when you're
> ready.
> 
> —
> Reply to this email directly, view it on GitHub
> [https://github.com/Technigo/project-portfolio/pull/406#pullrequestreview-2836337222],
> or unsubscribe
> [https://github.com/notifications/unsubscribe-auth/BKPH7WPLGEUZGBNWXAZTUXL26HIZ7AVCNFSM6AAAAABXX55U62VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQMZWGMZTOMRSGI].
> You are receiving this because you authored the thread.Message ID:
>***@***.***>

Envoyé avec la messagerie d'Infomaniak [https://ik.me], The Ethical
Cloud.

HIPPIEKICK commented:
See this screenshot of how it looks for me (I have dark mode turned on, that's why I asked you to try it in dark mode as well). If you don't want a specific dark mode styling you can remove that part from your CSS.
<img width="1920" alt="Screenshot 2025-05-22 at 11 53 31" src="https://github.com/user-attachments/assets/a9bce3e7-0586-49d2-bb7f-db7e26f8207a" />


Responsiveness looks better! Just make sure that your projects get the same padding as your articles, see images below:

<img width="598" alt="Screenshot 2025-05-22 at 11 56 39" src="https://github.com/user-attachments/assets/a5f609d9-77cb-4dcb-83d7-76edbea200c6" />


<img width="539" alt="Screenshot 2025-05-22 at 11 57 26" src="https://github.com/user-attachments/assets/6e071b14-c65a-484c-8e7e-41519feed128" />

smily342 committed:
Final cleanup: dark mode fixes, contrast compliance, responsive padding

smily342 commented:
Final CSS cleanup and accessibility update:
I had misunderstood some part of the dark mode comment
- Fixed all contrast issues flagged by Lighthouse (including <h3>, .role, and .skillsCategory items)
- Removed temporary debug styles (neon colors, outlines)
- Ensured consistent padding between .project and .post sections for visual coherence
- Improved dark mode support using prefers-color-scheme media queries
- Confirmed 100% accessibility score in Lighthouse

Ready for final review and merge.

smily342 committed:
Fix contrast issues and confirm padding layout

smily342 committed:
trying

smily342 committed:
Fix Skills section layout and dark-mode accessibility issues

smily342 commented:
fixing light and dark mode issues
padding and accessibility

smily342 committed:
accessibility resolution

smily342 commented:
To resolve contrast and accessibility issues for both light and dark modes, I had to slightly alter the original design.
accessibility is 100%

smily342 committed:
fix: final padding adjustments for layout polish
